make_check_sums.py

```diff
@@ -6,13 +6,17 @@
 
 
 def main():
-    files = [f for f in Path().cwd().rglob("*") if f.parent.name != "raven-testdata"]
+    files = [
+        f
+        for f in Path().cwd().rglob("*")
+        if f.parent.name != "raven-testdata" and not str(f).endswith(".md5")
+    ]
     for f in files:
-        outf = f'{f.as_posix()}.md5'
+        outf = f"{f.as_posix()}.md5"
         if f.is_file() and not Path(outf).exists():
```

Comment:
I suggest you review this logic instead. Ie remplace the file suffix by md5, then check if it exists.